client/lib/mixins/data-observe/README.md

```diff
@@ -10,7 +10,7 @@ A React mixin that makes it easy to trigger re-rendering of a component when a `
 ```js
```

Comment:
😱  As long as you're touching this README, how about a deprecation _"never **ever** use this"_ notice 😅 ?

Can this be used in non-mixin form? If so, it would be nice to provide an example and get rid of the `createClass` which we're about to drop 😁

Reply:
> Can this be used in non-mixin form? If so, it would be nice to provide an example and get rid of the `createClass` which we're about to drop 😁 

I think this has been discussed a couple of times, but I don't think it's worth spending time on implementing something like that and refactoring code to use it when we really want to use Redux.

And even if it were useful as a stop-gap (to facilitate getting rid of mixins), the README should still discourage use for newly written components.